CMakeLists.txt

```diff
@@ -136,9 +136,20 @@ message("Webmin: ${SSG_PRODUCT_WEBMIN}")
 
 message(" ")
 
+include(SSGCommon)
+
+# Targets 'validate' and 'zipfile' need to be added before any product because
+# they will receive dependencies from products added
+
 add_custom_target(validate)
```

Comment:
I'm a bit confused by  "As per the comment?"
The include will add the build macros but not the product targets itself. 

I just changed the order between `include(SSGCommon)` and `add_custom_target(validate)` because both `validate` and `zipfile` targets receive dependencies, and I think they should be one after another. And the include cannot go after `ssg_build_zipfile`.